csharp/NEP17/NEP17.cs

```diff
@@ -20,13 +21,16 @@ public partial class NEP17Demo : Nep17Token
         public static readonly StorageMap ContractMap = new StorageMap(Storage.CurrentContext, Prefix_Contract);
         private static readonly byte[] ownerKey = "owner".ToByteArray();
         private static bool IsOwner() => Runtime.CheckWitness(GetOwner());
-        public override byte Decimals() => 8;
+        public override byte Decimals() => Factor();
         public override string Symbol() => "NEP17";
 
+        public static byte Factor() => 8;
```

Comment:
Why two methods for the same value?

Reply:
> Why two methods for the same value?

Can't put Decimals into _deploy() since it's non-static.